Makefile

```diff
@@ -202,4 +202,4 @@ pristine: clean # also clean all canonical data and libwayne
 
 clean_canon_maps:
 	/bin/rm -f canon_maps/*[3-7].* # don't remove 8 since it takes a few minutes to create
-	/bin/rm -f orca_jesse_blant_table/UpperToLower*.txt
+	/bin/rm -f orca_jesse_blant_table/UpperToLower*.txt
```

Comment:
Please avoid making "null" changes--your editor has removed the blank line at the end of this file, and git marks it as a change. The blank line needs to be there, so this change should not be here. Arrange for your editor to stop removing trailing blank lines.